.github/workflows/ci.yml

```diff
@@ -25,6 +21,12 @@ jobs:
           cache: "yarn"
           node-version: "16"
 
+      # Inject all .env.example environment variables in $GITHUB_ENV
+      - name: "Set dotenv"
+        uses: c-py/action-dotenv-to-setenv@v2
+        with:
+          env-file: .env.example
```

Comment:
This one is interesting.

On one side I also don't like having ❌ in the PRs of this repo.

On the other, this is meant to be a template and having a `cp .env.example .env` in the middle of the Github action is a bit hidden for a developer starting a new project from this repo.

Reply:
As the ci is just using commands like yarn coverage and test so in my opinion it's safe to use .env.example for the action file. Once the user creates a repo locally they can definitely create their own .env for them and still use this .env.example for the ci.yml action file which is completely safe.

Do you have any suggestion to make this better?

Reply:
I agree with @ahmedali8, better to make the coverage script pass the first time. The user won't be able to use the template locally anyway, unless they eventually set a mnemonic.